website/forms/features/single-submit.md

```diff
@@ -0,0 +1,25 @@
+---
+id: single-submit
+title: Single submit
+---
+
+Single submit is a feature that allows you to submit a form when user makes any change on the form. This is useful for forms that require users to perform calculations, such as a mortgage calculator or a loan calculator.
+
+![Single submit](/img/forms/single-submit.webp)
+
+You can turn on single in the settings of the form.
+
+Single submit form can work with and without a submit button, but in general it is recommended not to use a submit button with single submit forms.
+
+Not all form field are supported by single submit. The following fields are supported:
+* Input range
+* Checkbox
+* Radio
+* Rating
+* Select
+
+If you add any of the non-supported fields to a single submit form, the form will not submit when the user makes a change to the non-supported field.
+
+:::caution
+This feature may create a large number of request to your server. Use with caution!
+:::
```

Comment:
```suggestion
Single submit allows continuous form submissions after the form data is added or changed, without the need to click a _Submit_ button.

![Single submit](/img/forms/single-submit.webp)

A good use case for this option are Calcualtor forms.

:::note
Even though it is supported, it is **not** recommended to have _Submit_ buttons within single submit forms.
:::

Also, not all fields are supported by single submit. The following fields are supported:
* Input range
* Checkbox
* Radio
* Rating
* Select

:::caution
If a non-supported field is added to a form with Single submit enabled, the feature will be disabled and the forms will not be auto-submitted on user input.
:::

:::caution
This feature can cause a large number of requests to be sent to the server. Use with caution!
:::
```

Also, consider renaming this to something like "Auto-submit", as _Single submit_ sounds like you can only submit it once 😄